test/src/kaltura-player.spec.js

```diff
@@ -89,8 +89,8 @@ describe('kaltura player api', function () {
           errorEventTriggered = true;
         });
         kalturaPlayer.loadMedia({}).catch(error => {
-          error.data.should.equal('Missing mandatory parameter');
-          error.success.should.be.false;
+          error.data.data.should.equal('Missing mandatory parameter');
```

Comment:
why this being changed? `data.data` seems weird also this is breaking change of the loadMedia catch listeners